### PR TITLE
Update severance policy

### PR DESCRIPTION
**Pull request by marbeann**
- Added our process for termination
- Added policy for voluntary resignation

Ref: https://github.com/teamniteo/operations/issues/1738

### CODE REVIEW

**Review by dmurko — APPROVED** [review]

File: 5_People/career.md [hunk 1]
```diff
@@ -102,13 +102,21 @@ Everyone tracks working hours for themselves. At the end of each month, add your
 
 ## Severance Policy
 
+### Termination
+
 While many of us are good friends, we are still a business, not a family. Unlike a family, people join and leave us from time to time. Whenever we have to let someone go, this is how we do it.
 
-* The person needs to know at least 1 month in advance that their position could be canceled in the near term.
+* The person is given three notices, the third one being the final chance. There should be minimum one month in between notices, approved by one founder. A notice expires after six months.
```
Comment (myrheimb) [inline 5_People/career.md:109]:
```suggestion
* The person is given three notices, the third one being the last chance. There should be a minimum of one month between notices, approved by one founder. A notice expires after six months.
```

File: 5_People/career.md [hunk 2]
```diff
@@ -102,13 +102,21 @@ Everyone tracks working hours for themselves. At the end of each month, add your
...
+* The person needs to be fully aware that they are being issued a notice. These are documented on their [catchup note](https://github.com/teamniteo/handbook/blob/master/5_People/catchups.md).
+* On second notice, [productivity allowance](https://github.com/teamniteo/handbook/blob/master/5_People/benefits.md#productivity-allowance) is capped at €100 (any amount larger needs to be approved). The immediate supervisor should prepare and discuss with the Nitean the steps/tasks that they need to do to pick up their performance.
```
Comment (myrheimb) [inline 5_People/career.md:111]:
```suggestion
* On the second notice, [productivity allowance](https://github.com/teamniteo/handbook/blob/master/5_People/benefits.md#productivity-allowance) is capped at €100. Any amount larger needs to be approved. The immediate supervisor should prepare and discuss with the Nitean the steps they need to take to fix the reason for the notice.
```
I also think we need to specify `capped at €100`.
Like, `capped at €100 per purchase` or `capped at €100 per month`.

File: 5_People/career.md [hunk 3]
```diff
@@ -102,13 +102,21 @@ Everyone tracks working hours for themselves. At the end of each month, add your
...
 * All Partners need to agree with the cancellation.
 * When cancellation is agreed, the person should be notified immediately by the immediate supervisor (usually the one doing the onboarding or has the most interaction with). From this point onwards, the person is no longer required to work, but is encouraged to take their time to hand off to other people in Niteo anything they feel is valuable. After hand-off is finished, they are free to do whatever.
 * Every Nitean (except Trialists) is entitled to a severance package paid in monthly payouts according to the following formula:
   * up to one year permanent: 1 month base salary,
   * more than one year permanent: 2 months base salary + 1 month for each additional year, capped at 6 months,
   * base salary is capped at 80% of level 3 salary which is the same for parental and sick leave.
+
+### Resignation
+
+When a resignation is tendered, [Benefits](https://github.com/teamniteo/handbook/blob/master/5_People/benefits.md) are automatically canceled. There is a minimum 30 days and max 60 days for turnover period.
```
Comment (myrheimb) [inline 5_People/career.md:122]:
```suggestion
When a resignation is tendered, [Benefits](https://github.com/teamniteo/handbook/blob/master/5_People/benefits.md) are automatically canceled. There is a minimum of 30 days and a maximum of 60 days for the turnover period.
```

File: 5_People/career.md (hunk 2 above)
Comment (dmurko) [reply 5_People/career.md:111]: The regular allowance is €135, so I don't think we meant per month. Maybe total or per purchase (so you can't buy a computer for 2k).

File: 5_People/career.md (hunk 1 above)
Comment (zupo) [reply 5_People/career.md:109]: If a notice expires after six months, and three are required before termination, then there isn't any room there, is it? Three notices spanning 2 months, already the first one expires?

Comment (dmurko) [reply 5_People/career.md:109]: Maybe 

> Notices expire a year after the last one was given.

File: 5_People/career.md (hunk 3 above)
Comment (zupo) [reply 5_People/career.md:122]: handover?

**Comment by zupo** [comment]
So, what's the status on this one? What remains to be polished?

**Pull request merged**